akka-persistence/src/main/scala/akka/persistence/snapshot/SnapshotStore.scala

```diff
@@ -81,7 +89,7 @@ trait SnapshotStore extends Actor {
    * @param metadata snapshot metadata.
```

Comment:
while you are here, can you add explicit return type to `saved` (and any other methods in the api)